packages/@aws-cdk/core/lib/cfn-output.ts

```diff
@@ -59,6 +59,66 @@ export class CfnOutput extends CfnElement {
     this._export = props.exportName;
   }
 
+  /**
+   * Returns the description of this Output
+   */
+  public getDescription(): string | undefined {
```

Comment:
All of these should use TypeScript-style getters and setters: https://www.typescriptlang.org/docs/handbook/classes.html#accessors